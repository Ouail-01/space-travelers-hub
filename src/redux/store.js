import { configureStore } from '@reduxjs/toolkit';
import rocketsSlice from './rockets/rocketsSlice';

const store = configureStore({
  reducer: {
    rockets: rocketsSlice,
    // categories: categoryReducer,
  },
});

export default store;
