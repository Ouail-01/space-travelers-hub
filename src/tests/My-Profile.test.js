import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import MyProfile from '../components/My-Profile';
import store from '../redux/store';

it('snapshot test for my profile page', () => {
  const myProfile = renderer.create(
    <Provider store={store}>
      <MyProfile />
    </Provider>,
  );
  const tree = myProfile.toJSON();
  expect(tree).toMatchSnapshot();
});
