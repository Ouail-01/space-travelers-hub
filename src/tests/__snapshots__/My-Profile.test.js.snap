// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test of profile page snapshot test for my profile page 1`] = `
<div
  className="profile-page"
>
  <section
    className="profile-rocket-section"
  >
    <h2
      className="profile-rocket-header"
    >
      My Rockets
    </h2>
    <h2
      className="no-reserved-rockets"
    >
      You have no reserved rockets!
    </h2>
  </section>
</div>
`;
